app/src/main/java/com/softvision/basiccalculator/android/cognizantcalculator/CalculatorActivity.kt

```diff
@@ -5,6 +5,7 @@ import android.os.Bundle
 import android.widget.Button
 import android.widget.TextView
 import android.widget.Toast
+import android.widget.Toast.LENGTH_LONG
```

Comment:
Delete this line and use Toast.LENGTH_SHORT instead on Line 52

Reply:
Done